ClickHouse.Ado/ClickHouseCommand.cs

```diff
@@ -18,15 +18,13 @@ public class ClickHouseCommand
         : IDbCommand
 #endif
     {
-        private ClickHouseConnection _clickHouseConnection;
```

Comment:
Can you revert this change? It can be problematic to merge changes from original repository in future due to conflicts